src/styles/search.css

```diff
@@ -0,0 +1,50 @@
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&family=Sofia+Sans:wght@200&display=swap");
+
+.search-input {
+  width: 300px;
+  height: 50px;
+  border-radius: 11px;
+  font-size: 25px;
+  padding: 0 10px;
+  background-color: rgb(50, 56, 75);
+  opacity: 0.9;
+  font-family: "Sofia Sans", sans-serif;
+  font-weight: bold;
+  color: white;
+  text-shadow: rgb(199, 180, 180) 1px 2px 2px;
+}
+
+.search-form {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  padding-top: 50px;
+}
+
+.submit-button {
+  color: rgb(255, 255, 255);
+  background-color: rgb(50, 56, 75);
+  opacity: 0.9;
+  border-radius: 8px;
+  font-size: 25px;
+  font-family: "Sofia Sans", sans-serif;
+  font-weight: bold;
+  text-shadow: rgb(199, 180, 180) 1px 1px 2px;
+  padding: 10px;
+}
```

Comment:
You could refactor this a bit. There are several properties which are the same in the `search-input` and `submit-button` classes, these could be combined to remove the duplication:
```
.search-input,
.submit-button {
  opacity: 0.9; 
  font-weight: bold;
  font-size: 25px;
  font-family: "Sofia Sans", sans-serif;
  text-shadow: rgb(199, 180, 180) 1px 2px 2px;
}

.search-input {
  width: 300px;
  height: 50px;
  border-radius: 11px;
  padding: 0 10px;
  background-color: rgb(50, 56, 75);
  color: white;
}

.search-form {
  display: flex;
  gap: 10px;
  padding-top: 50px;
}

.submit-button {
  color: rgb(255, 255, 255);
  background-color: rgb(50, 56, 75);
  border-radius: 8px;
  padding: 10px;
}
```

Also, you could remove the flex-direction: row property from `search-form` as flex-direction is implicitly row already, you dont need to declare it unless youre overwriting an earlier declaration